core/src/main/java/org/apache/accumulo/core/fate/WrappedFateTxStore.java

```diff
@@ -86,6 +86,12 @@ public void delete() {
     wrapped.delete();
   }
 
+  @Override
+  public void forceDelete() {
+    throw new UnsupportedOperationException(
```

Comment:
I like what this trying to do, but its far removed from the code that cares about this and if the code is refactored then this validation could be lost w/o anyone realizing it.   

The following is a not a great solution, but its a solution.  Could pass a boolean to the WrappedFateTxStore that indicates if forceDelete should be allowed or not.  This boolean could then cascade out to the code that creates the logging fate store.  The boolean would always be false, but it links the code that really cares about this to this check and would force any refactorings to consider it.

Another solution is to add a comment exlaining why this throwing unsupported operation exception.

Reply:
Yeah, good point... I feel like comments are also likely to be overlooked/forgotten. I added the boolean in 9069a33534279929c3c42e08a1a30834cbbda473. It seems like a fine solution.

Reply:
Those changes look good.